### PR TITLE
Split service

### PR DESCRIPTION
**Pull request by joecks**
Splits the service in a generic and a specific part.
Used for the embedding service
Also fix missing profile var in the invoke script

### CODE REVIEW

File: b2b/provisioner/elasticsearch_index/variables.tf [hunk 1]
```diff
@@ -33,3 +33,8 @@ variable "skip_delete" {
   type        = bool
   default     = true
 }
+
+variable "aws_profile" {
+  description = "AWS Profile with witch the provisioner should be executed"
+  type        = string
+}
```
Comment (Robert-Steiner) [inline b2b/provisioner/elasticsearch_index/variables.tf:40]:
```suggestion
}

```

File: b2b/provisioner/elasticsearch_index/variables.tf [hunk 2]
```diff
@@ -33,3 +33,8 @@ variable "skip_delete" {
   type        = bool
   default     = true
 }
+
+variable "aws_profile" {
+  description = "AWS Profile with witch the provisioner should be executed"
```
Comment (Robert-Steiner) [inline b2b/provisioner/elasticsearch_index/variables.tf:38]:
```suggestion
  description = "AWS Profile with which the provisioner should be executed"
```

File: b2b/provisioner/postgres_database/variables.tf [hunk 3]
```diff
@@ -33,3 +33,8 @@ variable "skip_delete" {
   type        = bool
   default     = true
 }
+
+variable "aws_profile" {
+  description = "AWS Profile with witch the provisioner should be executed"
+  type        = string
+}
```
Comment (Robert-Steiner) [inline b2b/provisioner/postgres_database/variables.tf:40]:
```suggestion
}

```

File: b2b/provisioner/postgres_database/variables.tf [hunk 4]
```diff
@@ -33,3 +33,8 @@ variable "skip_delete" {
   type        = bool
   default     = true
 }
+
+variable "aws_profile" {
+  description = "AWS Profile with witch the provisioner should be executed"
```
Comment (Robert-Steiner) [inline b2b/provisioner/postgres_database/variables.tf:38]:
```suggestion
  description = "AWS Profile with which the provisioner should be executed"
```

Comment (joecks) [reply b2b/provisioner/postgres_database/variables.tf:38]: I like Witches more than Whiches :P

**Review by Robert-Steiner — APPROVED** [review]
nice refactoring 👍

**Pull request merged**